Lib/test/test_concurrent_futures.py

```diff
@@ -1018,6 +1017,39 @@ def test_idle_process_reuse_multiple(self):
         self.assertLessEqual(len(executor._processes), 2)
         executor.shutdown()
 
+    def test_max_tasks_per_child(self):
+        executor = self.executor_type(1, max_tasks_per_child=3)
+        f1 = executor.submit(mul, 6, 7)
+        self.assertEqual(f1.result(), 42)
+        self.assertEqual(len(executor._processes), 1)
+        original_pid = [pid for pid in executor._processes.keys()][0]
```

Comment:
Instead of peeking in the executor internals, it may be nicer to just submit `os.getpid` on the pool, no?

Reply:
I forgot to delete `_spawn_replacement_process`. My apologies.

yes, we can use `os.getpid` to test the same thing.